pkg/lib/ct-card.scss

```diff
@@ -29,6 +29,23 @@
   padding: var(--pf-global--spacer--md);
 }
 
+// Strip margins, especially negative margins, but keep expansion
+.ct-card.pf-c-card .pf-c-card__actions {
+  margin: 0 0 0 auto;
+}
+
+// Remove redundant padding from embedded toolbars (handled by header)
+.ct-card.pf-c-card .pf-c-toolbar {
+  padding: 0;
+}
+
+// Remove excess top padding from empty state in cards,
+// as card headers already add enough space
+.ct-card.pf-c-card .pf-c-empty-state {
+    --pf-c-empty-state__body--MarginTop: 0;
+    padding-top: 0;
```

Comment:
This looks wrong in another pages now.
![Screen Shot 2021-03-29 at 17 03 57](https://user-images.githubusercontent.com/14921356/112857245-f55ba580-90b0-11eb-977c-93198078f0cc.png)
![Screen Shot 2021-03-29 at 17 03 26](https://user-images.githubusercontent.com/14921356/112857248-f5f43c00-90b0-11eb-9a89-ab62e79fc0d2.png)

Reply:
Ugh. This is broken... but...

- Those pages should be using an empty state directly instead of a list with an empty state.
- List with empty states are supposed to be used for filters on a list, and generally contain reset the list as a pattern.

I guess we could change the CSS to affect direct empty states instead of all empty states. (And fix the lists in another PR as they're out of scope here.)

Thanks for pointing this out. I'll fix.

Reply:
I think we can do it in this PR, it's literally one small place to change it.

Reply:
And then I don't need to handle explicitely the NFS case.

Reply:
Sounds good. I'll still make the padding/margin changes only apply to a toplevel empty state in the CSS meanwhile.

Having empty states in lists still makes sense for when they're being filtered. I'd hate to break that for any lists in cards that are filtered.